external-crates/move/tools/move-package/tests/test_sources/parsing_minimal_manifest/Move.resolved

```diff
@@ -9,10 +9,8 @@ ResolvedGraph {
         always_deps: {
             "name",
         },
-        manifest_digest: Some(
-            "40ECD99EA24DDE388D25DBAB6EFAAF04543D55F0AD108BA6F489B0C71BB618AB",
-        ),
-        deps_digest: None,
+        manifest_digest: "40ECD99EA24DDE388D25DBAB6EFAAF04543D55F0AD108BA6F489B0C71BB618AB",
+        deps_digest: "",
```

Comment:
I'm guessing we get an empty string here because there are no deps -- thoughts on making this the digest of the empty list instead?  I'm not wedded to it, but I'm trying to anticipate difficulties in interpreting this as a digest elsewhere (in other tools) if it could also be an empty string.

Reply:
Indeed! And I don't have a strong opinion here so made it a digest of an empty list